src/components/MyKiva/BadgeTile.vue

```diff
@@ -6,13 +6,13 @@
 		/>
 		<div
 			v-else
-			class="tw-cursor-pointer"
+			class="tw-cursor-pointer md:tw-w-1/2"
```

Comment:
The request from design was just the white box, so the click and cursor should be moved to the div under the h4.